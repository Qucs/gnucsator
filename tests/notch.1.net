# Qucs 0.1.0  incomplete
## declarations end

C:C1 _net9 _net15 C="1.01u" V=""
R:R5 In Out R="10k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R3 _net10 _net18 R="1k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R4 _net5 _net18 R="1k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R2 _net15 _net10 R="1k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
C:C2 _net5 Out C="1u" V=""
OpAmp:OP1 _net5 _net9 _net15 G="1e6" Umax="15 V"
OpAmp:OP2 _net5 _net10 _net18 G="1e6" Umax="15 V"
# Eqn:Eqn1 Gain="dB(Out.v/In.v)" Export="yes"
R:R1 gnd _net9 R="10k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
Vac:V1 In gnd U="1uV" f="1 GHz" Phase="0" Theta="0"
.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
.AC:AC1 Type="log" Start="5" Stop="500" Points="10" Noise="no"
