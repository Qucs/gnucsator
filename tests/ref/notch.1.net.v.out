
C #(.C(1.01u)) C1 (.p(_net9),.n(_net15));
R #(.R(10k),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R5 (.p(In),.n(Out));
R #(.R(1k),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R3 (.p(_net10),.n(_net18));
R #(.R(1k),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R4 (.p(_net5),.n(_net18));
R #(.R(1k),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R2 (.p(_net15),.n(_net10));
C #(.C(1u)) C2 (.p(_net5),.n(Out));
OpAmp #(.G(1e6),.Umax(15)) OP1 (.1(_net5),.2(_net9),.3(_net15));
OpAmp #(.G(1e6),.Umax(15)) OP2 (.1(_net5),.2(_net10),.3(_net18));
R #(.R(10k),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R1 (.p(0),.n(_net9));
Vac #(.Phase(0),.Theta(0),.U(1uV),.f(1G)) V1 (.1(In),.2(0));
